publish-subscribe/subscriber.bal

```diff
@@ -0,0 +1,33 @@
+import ballerinax/kafka;
+import ballerina/uuid;
+import ballerina/websocket;
+
+service /ws on new websocket:Listener(8081) {
+
+    resource function get [string matchId]() returns websocket:Service|error {
+        return new MatchBroadcastService(matchId);
+    }
+}
+
+isolated service class MatchBroadcastService {
+    *websocket:Service;
+    private final string matchId;
+
+    public function init(string matchId) {
+        self.matchId = matchId;
+    }
+
+    isolated remote function onOpen(websocket:Caller caller) returns error? {
+        final kafka:Consumer kafkaConsumer = check new (kafka:DEFAULT_URL, {
+            groupId: string `realtime-web-ui-group-${uuid:createType1AsString()}`,
+            topics: [self.matchId]
+        });
+        while true {
+            [string, record {}][] matchDetails = check kafkaConsumer->pollPayload(1);
```

Comment:
what is this string that is getting ignored ?

Reply:
match id.